src/utils/logging.jl

```diff
@@ -1,6 +1,14 @@
 
 import Logging
 
+# These use PascalCase to avoid clashing with source filenames.
+const LOG_GROUP_RECORDER = :Recorder
+const LOG_GROUP_SERIALIZATION = :Serialization
+const LOG_GROUP_SYSTEM = :System
+const LOG_GROUP_TIME_SERIES = :TimeSeries
```

Comment:
I think it would be good to use a convention like this to help standardize.

Reply:
Is it possible to define other groups later for PSI in PSI or we need to define them here?

Reply:
We should try to put the common ones here, but I think it would be too onerous to make that a rule. Each package could define its own as needed using a similar convention.